### PR TITLE
📝 Documentation for warehousing

### PR DESCRIPTION
**Pull request by fiendish**

### CODE REVIEW

File: docs/source/design/warehousing.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+your package's Extract, Transform, and Load results as well as the KFID lookup
+cache for the study. This makes it easier for others to assist in replicating,
+validating, and modifying the results of an ingest. Sharing is caring!
+
+The Warehouse Server
+====================
+
+You will need:
+
+1. a database server that supports namespace schemas (ask your group
```
Comment (znatty22) [inline docs/source/design/warehousing.rst]:
```suggestion
1. A database server that supports namespace schemas (ask your group
```

File: docs/source/design/warehousing.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+   administrator if one has already been set up)
+
+   * PostgreSQL will work
+   * MySQL will **not** work
+
+2. an account on the database server with sufficient access permissions to
```
Comment (znatty22) [inline docs/source/design/warehousing.rst]:
```suggestion
2. An account on the database server with sufficient access permissions to
```

File: docs/source/design/warehousing.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+   invoke ``CREATE DATABASE``, ``CREATE SCHEMA``, ``DROP SCHEMA``,
+   ``CREATE TABLE``, ``DROP TABLE``, and ``INSERT``
+
+3. the power of friendship
```
Comment (znatty22) [inline docs/source/design/warehousing.rst]: Lol uhmm...

File: docs/source/design/warehousing.rst [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+Local Environment Configuration
+===============================
+
+In your local environment, set a variable called ``KF_WAREHOUSE_DB_URL``
+containing a complete login URL, including username and password if needed,
+that can be used for calling ``CREATE DATABASE`` on the server, like this::
+
+    export KF_WAREHOUSE_DB_URL=postgresql://<username>:<password>@<address>:<port>/postgres
```
Comment (znatty22) [inline docs/source/design/warehousing.rst]:
```suggestion
    export KF_WAREHOUSE_DB_URL=postgresql://<username>:<password>@<ip address>:<port>/postgres
```

File: docs/source/design/warehousing.rst [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+after the study KFID.
+
+.. image:: /_static/images/warehouse_db.png
+
+Inside that database you should find schemas for each of the warehoused stages
+prefixed by "Ingest:" and the name of the package. Each schema table will be
+one of that stage's returned DataFrames.
+
+.. image:: /_static/images/warehouse_inside_db.png
+
```
Comment (znatty22) [inline docs/source/design/warehousing.rst]:
```suggestion
Details for the warehouse used for ingesting Kids First studies can be found [here](https://github.com/d3b-center/clinical-data-flow/tree/master/warehouse)
```

File: docs/source/_static/css/style.css [hunk 6]
```diff
@@ -1,31 +0,0 @@
-.wy-side-nav-search {
```
[the remark this answers is not in the document]
Comment (fiendish) [reply docs/source/_static/css/style.css]: As far as I can tell, we were not using it. It was an _almost_ clone of the kf theme css file that we weren't loading.

File: docs/source/design/warehousing.rst (hunk 4 above)
Comment (fiendish) [reply docs/source/design/warehousing.rst]: it could be a URL

**Comment by fiendish** [comment]
> 🐥

What is that. A yellow penguin?

**Comment by znatty22** [comment]
> > 🐥
> 
> What is that. A yellow penguin?

A cute little chick!

**Pull request merged**